components/speedreader/rust/ffi/Cargo.lock

```diff
@@ -145,6 +145,19 @@ version = "1.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
 
+[[package]]
```

Comment:
NOTE to reviewers: can just ignore this Cargo.lock. These dependency changes were already merged in another PR, and I forgot to bump the ffi Cargo.lock